docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -428,6 +429,10 @@ similar to `Environment`.
 
 This key can be listed multiple times.
 
+### `AppArmor=`
+
+Sets the apparmor confinement profile for the container. A value of `unconfined` turns off apparmor confinement.  (Default: true)
```

Comment:
When I first reviewed this, I could have sworn it was a boolean value.  But it looks like it takes a string?  If so, please retract my earlier comment to add the "(Default: true)".  If there is a default, please replace `true` with it, if not, just remove the whole parens here.